exporters/sender/jdk/src/main/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHtttpUtil.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.sender.jdk.internal;
+
+import io.opentelemetry.sdk.internal.DaemonThreadFactory;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.SynchronousQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Utilities for JDK HTTP.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public class JdkHtttpUtil {
```

Comment:
I think you are copying the pattern of OkHttpUtil in terms of having a utility class, but that utility exists because there are both gRPC and http/protobuf implementations that use have the same logic for creating a dispatcher. Can just keep this as a private helper function in `JdkHttpSender`.